Feat/main frame

sejung2 opened the pull request:
## Overview

- 앱 메인 프레임 초기구현

## Changes Made

- [x] 공통 타입 import 방식 변경(기존 구조 삭제 후, 패키지를 통해 연결)
- [x] 커스텀 타이틀 바 구현(닫기, 최소, 최대), 드래그 영역 설정
- [x] 전역 알림 시스템을 위한 Toaster 컴포넌트 추가
- [x] 사이드바 추가를 위한 구조 조정  

## Additional Notes

- 추가적으로 기능 보완 예정

sejung2 committed:
chore: 이전 설정 복구

sejung2 committed:
chore: common 패키지 추가

- 패키지 방식으로 기존 파일들 수정(dummySchedule.ts, calendarStore.ts)

sejung2 committed:
feat: implement main-frame ui

- 커스텀 윈도우바 초기 구현
- 추후 사이드바 통합을 위한 기반 마련

sejung2 committed:
feat: add toaster

- 알림 시스템을 위해 Toaster 컴포넌트 추가

bluenyang reviewed (changes requested):
copilot 제안 + 몇가지 수정사항 있습니다~

sejung2 committed:
fix: fix main frame style and delete default style(style.css)

- 윈도우바 아이콘으로 변경, 접근성을 위해 area-label 추가
- SidebarProvider 스타일 수정(style 제거 및 tailwindcss로 적용)
- 기본 스타일 제거(style.css 연결만 해제함)

sejung2 commented on app/renderer/src/App.vue:
```diff
@@ -1,63 +1,79 @@
+        <span class="text-xs font-bold text-[#4A4A4A]">Croffle</span>
+      </div>
+
+      <div class="no-drag flex gap-1">
```
반영 완료

bluenyang reviewed (approved):
LGTM
